startup_scripts/startup_script_utils/permissions.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.models import Permission
+
+
+def set_perms_from_yaml(subject=None, permissions=None):
+  if subject is None or permissions is None:
+    return
+  subject.clear()
+  for perm in permissions:
```

Comment:
Here too, I advise against using abbreviations if they don't add clarity. And I believe the abbreviation `perm` here does not add clarity over just writing `permission`.

```suggestion
  for permission in permissions:
```

Reply:
If I call that variable `permission` we have a conflict with line 17 where `permission` is used for the objects loaded from the database.
So I could change it to `permission_string`